eng/Packages.Data.props

```diff
@@ -71,6 +71,8 @@
     <PackageReference Update="System.ValueTuple" Version="4.5.0" />
     <PackageReference Update="Microsoft.Bcl.AsyncInterfaces" Version="1.0.0" />
     <PackageReference Update="Microsoft.CSharp" Version="4.5.0" />
+    <PackageReference Update="System.Collections.Immutable" Version="1.3.1" />
+    <PackageReference Include="System.Linq.Async" Version="4.0.0" />
```

Comment:
We should not have any Include's in this file, only updates. Having this be include will add it to all projects which we don't want.

Reply:
doh, good catch meant to be an update